src/containerapp/azext_containerapp/_params.py

```diff
@@ -446,3 +446,36 @@ def load_arguments(self, _):
         c.argument('resource_group_name', arg_type=resource_group_name_type)
         c.argument('managed_env', options_list=['--environment', '-e'], help='Name or resource id of the Container App environment.')
         c.argument('show_all', action='store_true', help='Show all patchable and unpatchable container apps')
+
+    # Container App job
+    with self.argument_context('containerapp job') as c:
+        c.argument('name', name_type, metavar='NAME', id_part='name', help=f"The name of the Container Apps Job. A name must consist of lower case alphanumeric characters or '-', start with a letter, end with an alphanumeric character, cannot have '--', and must be less than {MAXIMUM_CONTAINER_APP_NAME_LENGTH} characters.")
+        c.argument('cron_expression', help='Cron expression. Only supported for trigger type "Schedule"')
+        c.argument('image', help="Container image, e.g. publisher/image-name:tag.")
+        c.argument('replica_completion_count', type=int, options_list=['--replica-completion-count', '--rcc'], help='Number of replicas that need to complete successfully for execution to succeed.')
+        c.argument('replica_retry_limit', type=int, help='Maximum number of retries before the replica fails.')
+        c.argument('replica_timeout', type=int, help='Maximum number of seconds a replica can execute.')
+        c.argument('parallelism', type=int, help='Maximum number of replicas to run per execution.')
+        c.argument('workload_profile_name', options_list=['--workload-profile-name', '-w'], help='The friendly name for the workload profile')
+        c.argument('min_executions', type=int, help="Minimum number of job executions that are created for a trigger, default 0.")
+        c.argument('max_executions', type=int, help="Maximum number of job executions that are created for a trigger, default 100.")
+        c.argument('polling_interval', type=int, help="Interval to check each event source in seconds. Defaults to 30s.", default=30)
+
+    with self.argument_context('containerapp job create') as c:
+        c.argument('system_assigned', help='System assigned identity.', action='store_true')
+        c.argument('trigger_type', help='Trigger type. Schedule | Event | Manual')
+        c.argument('user_assigned', help='User assigned identity.')
```

Comment:
```suggestion
        c.argument('mi_system_assigned', help='System assigned identity.', action='store_true')
        c.argument('trigger_type', help='Trigger type. Schedule | Event | Manual')
        c.argument('mi_user_assigned', help='User assigned identity.')
```

Reply:
The existed 'containerapp create' use these params too, hope to be consistent with the previous:
```
    with self.argument_context('containerapp create', arg_group='Identity') as c:
        c.argument('user_assigned', nargs='+', help="Space-separated user identities to be assigned.")
        c.argument('system_assigned', help="Boolean indicating whether to assign system-assigned identity.")
```

Reply:
In a future breaking change window, these parameters could be migrated together into the new parameters formulated by our specifications in a unified PR.